### ISSUE TITLE
vessel-init: Auto-choose custom Docker image

### ISSUE THREAD

**Issue by mtlynch** [Vessel-App/vessel-cli#24]
This is just a feature request, but in `vessel init`, the flow is a little surprising. It prompts the user to specify a Docker image, then it takes the user back to the list of Docker images and asks them to re-select the image they just specified. It would be a bit easier if vessel just auto-selected the image that the image that the user just specified.

**Comment by fideloper** [comment]
I agree, and perhaps that's possible! It's taking that behavior from https://github.com/manifoldco/promptui (so if the behavior is changeable it would be via that package, or a configuration I missed!)
